tests/integration/test_ethernet.py

```diff
@@ -0,0 +1,93 @@
+import os
+import pytest
+import subprocess
+import sys
+
+VETH_NAME = "testeth"
+
+if sys.version_info >=(3, 3):
+    from unittest import mock
+else:
+    import mock
```

Comment:
```suggestion
try:
    from unittest import mock
except ImportError:
    import mock
```

Reply:
IMO, this should go on the import section.